examples/model_selection/plot_det.py

```diff
@@ -0,0 +1,121 @@
+"""
+=======================================
+Detection error tradeoff (DET) curve
+=======================================
+
+In this example we compare receiver operating characteristic (ROC) and
+detection error tradeoff (DET) curves to demonstrate how DET curves can help
+to asses the performance of different classification algorithms.
+
+DET curves are commonly plotted in normal deviate scale.
+To achieve this we transform the errors rates as returned by the
+``detection_error_tradeoff_curve`` function and the axis scale using
+``scipy.stats.norm``.
```

Comment:
please add a few words on what this example demonstrates? what is the take home message? Are you saying that DET makes it easier to highlight that RF is better than ROC?

Reply:
This example merely tries to demonstrate to the unfamiliar user how an example of a DET curve can look like. I tried to not repeat myself too much, and the properties, pros and cons are already discussed in the usage guide (which uses the sample plot).
Would you be fine if I added an explicit link to the usage guide here too?